openy.install

```diff
@@ -741,7 +741,7 @@ function openy_update_8050() {
 /**
  * Enable CSS Editor module and import the settings.
  */
-function openy_update_8050() {
+function openy_update_8051() {
```

Comment:
@podarok Dmitriy has fixed this number because I missed it during the merge of CSS Editor module. Now all numbers are correct.